ldap/src/main/java/org/springframework/security/ldap/authentication/ad/DefaultActiveDirectoryAuthoritiesPopulator.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2024 the original author or authors.
```

Comment:
Even though this class was created this year, for consistency we still add the date range. Please change this to `2002-2024`.